content/en/blog/2023/sig-network-blixt-ebpf-rust-in-kubernetes.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: blog
+title: Blixt - A load-balancer written in Rust, using eBPF, born from Gateway API
+date: 2023-11-15
+slug: blixt-load-balancer-rust-ebpf-gateway-api
+author: "Shane Utt (Kong), Andrew Stoycos (Red Hat)"
+---
+
+In [SIG Network][signet] we now have a layer 4 load-balancer named [Blixt][blixt]. This
+project started as a fun experiment using emerging technologies and is intended
+to become a utility for CI and testing to help facilitate the continued
+development of [Gateway API][gwapi]. Are you interested in developing networking
+tools in [Rust][rust] and [eBPF][ebpf]? Or perhaps you're specifically
+interested in Gateway API? We'll tell you a bit about the project and how it
+might benefit you.
+
+[signet]:https://github.com/kubernetes/community/tree/master/sig-network
+[blixt]:https://github.com/kubernetes-sigs/blixt
+[gwapi]:https://github.com/kubernetes-sigs/gateway-api
+[rust]:https://www.rust-lang.org/
+[ebpf]:https://www.kernel.org/doc/html/latest/bpf/index.html
+
+## History
+
+[Blixt][blixt] originated at [Kong][kong] as an experiment to test
+load-balancing ingress traffic for Kubernetes clusters using eBPF for the
+dataplane. Around the time of Kubecon Detroit (2022) we (the [Gateway
+API][gwapi] maintainers) realized it had significant potential to help us move
+our `TCPRoute` and `UDPRoute` support forward, which had been sort of "stuck in
+alpha" at the time due to a lack of conformance tests being developed for them.
+At the same time various others in the SIG Network community developed an
+interest in the project due to the rapid growth of eBPFs use on Kubernetes.
+Given the potential for benefit to the Kubernetes ecosystem and the growing
+interest, Kong decided it would be helpful to [donate the project to Kubernetes
+SIGs][donation] to benefit upstream Kubernetes.
+
+Over several months we rewrote the project in [Rust][rust] (from C) due to a
+strong contingency of Rust knowledge (and interest) between us developing the
+project and an active interest in the burgeoning [Aya project][aya] (a Rust
+framework for developing eBPF programs). We did eventually move the
+control-plane (specifically) to [Golang][go] however, so that we could take
```

Comment:
```suggestion
control plane (specifically) to [Golang][go] however, so that we could take
```
For Kubernetes, "control plane" is usually not hyphenated. Though sometimes I've seen it hyphenated when referring to something like a "control-plane node".